samples/Sentry.Samples.AspNetCore.Grpc/GameService.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Data;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Google.Protobuf.WellKnownTypes;
+using Grpc.Core;
+using Microsoft.Extensions.Logging;
+using Sentry.AspNetCore.Samples.Grpc;
+
+namespace Sentry.Samples.AspNetCore.Grpc
+{
+    public class GameService : Games.GamesBase
+    {
+        private readonly ILogger _gameLogger;
+
+        public GameService(ILogger<GameService> gameLogger) => _gameLogger = gameLogger;
+
+        public override async Task<NextPhaseResponse> FetchNextPhaseData(Empty request, ServerCallContext context)
+        {
+            _gameLogger.LogInformation("Fetching dungeons and mana level in parallel.");
+
+            var getDungeonsTask =
+                Task.Run(new Func<int>(() => throw new HttpRequestException("Failed to fetch available Dungeons")));
+            var getUserMana = Task.Run(new Func<int>(() => throw new DataException("Invalid Mana level: -10")));
```

Comment:
Is specifying `Func<int>` necessary here?

Reply:
@Tyrrrz I was following how it was in the MVC sample, but yes. It needs to return a `Task<int>`. Attempting to do `Task.Run<int>(() => throw ..)` would result in an ambiguous invocation, so I can't do that either.